src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * This class contains functionality that deals with making sense of user input
+ */
+package duke.parser;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import duke.DukeException;
+import duke.exec.Add;
+import duke.exec.Delete;
+import duke.exec.Done;
+import duke.exec.Executable;
+import duke.exec.Exit;
+import duke.exec.Find;
+import duke.exec.Listing;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+public class Parser {
+
+    // Command constants for the bot
```

Comment:
I noticed that some of such comments are always lowercase while others such as this are in sentence case. I think it would be a good idea to keep to a particular format?